language/move-lang/tests/move_check/expansion/duplicate_alias.exp

```diff
@@ -3,7 +3,7 @@ error:
    ┌── tests/move_check/expansion/duplicate_alias.move:8:19 ───
    │
  8 │     use 0x1::Y as Z;
```

Comment:
This error message would make slightly more sense to me if it pointed to line 7 and "Previously defined here" pointed to line 8.

Reply:
Why? The definitions go in order. So the alias on line `7` is defined first, thus making it the previous definition

Reply:
If this is just about the ordering of the actual messages, and not the ordering of which is defined first, all of the error messages will be completely rebuilt to annotate in order of the line numbers (instead of the current order where labels are laid out in order they were added)

See #3447 for more detail about this future migration